src/umpire/device_allocator_helper.hpp

```diff
@@ -62,11 +62,13 @@ __host__ void destroy_device_allocator();
 #if defined(UMPIRE_ENABLE_CUDA)
 #define UMPIRE_SET_UP_DEVICE_ALLOCATORS()                                                                          \
   {                                                                                                                \
+    UMPIRE_LOG(Debug, "Calling cudaMemcpyToSymbol macro.");                                                        \
     cudaMemcpyToSymbol(umpire::UMPIRE_DEV_ALLOCS, &umpire::UMPIRE_DEV_ALLOCS_h, sizeof(umpire::DeviceAllocator*)); \
   }
 #elif defined(UMPIRE_ENABLE_HIP)
 #define UMPIRE_SET_UP_DEVICE_ALLOCATORS()                                                                         \
   {                                                                                                               \
+    UMPIRE_LOG(Debug, "Calling hipMemcpyToSymbol macro.");                                                        \
     hipMemcpyToSymbol(umpire::UMPIRE_DEV_ALLOCS, &umpire::UMPIRE_DEV_ALLOCS_h, sizeof(umpire::DeviceAllocator*)); \
```

Comment:
check for null ptr (see how desul does this) - don't want to have too many duplicate copy calls